addons/web/static/src/public/colibri.js

```diff
@@ -296,23 +312,25 @@ export class Colibri {
             throw new Error("Updatecontent should not be called while interaction is updating");
         }
         this.isUpdating = true;
+        this.refreshNodes();
         const errors = [];
         const interaction = this.interaction;
         for (const dynamicAttr of this.dynamicAttrs) {
-            const { nodes, attr, definition, initialValues } = dynamicAttr;
+            const { sel, attr, definition, initialValues } = dynamicAttr;
             let valuePerNode;
-            if (!initialValues) {
+            const nodes = this.dynamicNodes.get(sel);
```

Comment:
`|| []` here, then you don't have to worry about it in the `if` and `for` below ?